query-request.hh

```diff
@@ -230,8 +229,8 @@ public:
     const std::unique_ptr<specific_ranges>& get_specific_ranges() const {
         return _specific_ranges;
     }
-    const cql_serialization_format& cql_format() const {
-        return _cql_format;
+    const cql_serialization_format cql_format() const {
+        return cql_serialization_format(4); // For IDL compatibility
```

Comment:
I don't I understand why 4. We started with 3, not 4. Does it need to be specifically 4 for IDL compatibility?

Reply:
4 and 3 are equivalent in terms of the serialization format. Since most clients use protocol 4 now, I chose to write 4 here. New nodes will ignore it; old nodes will see 4 and use 32-bit serialization.